gitops/overlays/olivier/kustomization.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: canada-dental-care-plan
+nameSuffix: -olivier
+commonLabels:
+  # commonLabels must have at least one unique label
+  # per environment to ensure selectors are applied correctly
+  app.kubernetes.io/instance: olivier
+labels:
+  - pairs:
+      app.kubernetes.io/part-of: canada-dental-care-plan
+      app.kubernetes.io/managed-by: teamcity
+      app.kubernetes.io/cluster: dts-olivier-sced-rhp-spoke-aks
```

Comment:
This needs to be `dts-dev-sced-rhp-spoke-aks`. The cluster name is not set by us. :)

Sorry.. I know it is probably confusing.

Reply:
i switched it back to dev